<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopupSettings</class>
 <widget class="QDialog" name="PopupSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Popup Information Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="5">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Template</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0" colspan="3">
       <widget class="QPlainTextEdit" name="textEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>138</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="insertButton">
        <property name="text">
         <string>Insert</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QCheckBox" name="coverCheckBox">
     <property name="text">
      <string>Show cover</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="coverSizeLabel">
     <property name="text">
      <string notr="true">0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Cover size:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSlider" name="coverSizeSlider">
     <property name="minimum">
      <number>32</number>
     </property>
     <property name="maximum">
      <number>128</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>16</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Transparency:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QSlider" name="transparencySlider">
     <property name="maximum">
      <number>90</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="transparencyLabel">
     <property name="minimumSize">
      <size>
       <width>18</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string notr="true">0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Delay:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QSpinBox" name="delaySpinBox">
     <property name="minimum">
      <number>20</number>
     </property>
     <property name="maximum">
      <number>7000</number>
     </property>
     <property name="singleStep">
      <number>500</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PopupSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PopupSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
